pkgs/by-name/je/jellyfin-web/package.nix

```diff
@@ -4,7 +4,6 @@
   overrideSDK,
   fetchFromGitHub,
   buildNpmPackage,
-  jellyfin,
```

Comment:
Oops I accidentally removed `assert version == jellyfin.version;` in https://github.com/NixOS/nixpkgs/pull/351966/files#diff-627fe354cb06fef65417ca768bd1a60e83edfa7b20551aed65a3a0d4facd6452L33. Maybe we should add it back?

Reply:
Wasn't sure if that was intentional, i can add it back.